packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-transfer.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Movement } from '@carbon/react/icons';
+import type { WardPatientCardElement } from '../../types';
+import styles from './row-elements.scss';
+import { useTranslation } from 'react-i18next';
+
+const WardPatientTransfer: WardPatientCardElement = () => {
+  const { t } = useTranslation();
+  // TODO update the transfer details by the active disposition
+  return (
+    <div className={styles.waitingForItemContainer}>
+      <Movement className={styles.movementIcon} size="24" />
+      {t('transferToDeliveryWard', 'Transfer to Delivery ward')}
+    </div>
```

Comment:
This is just a more or less a placeholder at this point, correct?

Reply:
It looks like it. @usamaidrsk ?

Reply:
Oh interesting, I missed let, I will check out ticket and comment....

Reply:
Okay @usamaidrsk , I updated the ticket, you are right, we can't display the TRANSER or DISCHARGE until we finish https://openmrs.atlassian.net/browse/O3-3648 , which I hope to start on shortly.